### PR TITLE
Improve .gitignore & FAQ

### PR DESCRIPTION
**Pull request by melroy89**
- Add the file `public/media` to `.gitignore` as well, since I use a symbolic link under Linux now for the media directory. Which is seen as "a file".
- FAQ changed due to PR merged: https://github.com/MbinOrg/mbin/pull/208

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -17,6 +17,7 @@ tools/vendor/
 # Mbin specific
 .env
 /public/media/*
+/public/media
```
Comment (e-five256) [inline .gitignore:20]: I think you could also do a single rule of `/public/media*` to cover both situations but that is a minor nitpick

File: FAQ.md [hunk 2]
```diff
@@ -124,10 +124,12 @@ That said, if you are _experiencing serious issues_ with your instance which you
...
 ## I changed my .env configuration but the error still appears/new config doesn't seem to be applied?
 
-After you edited your `.env` configuration file on a bare metal/VM setup, you always need to execute the dump-env command manually (this is **not** needed on Docker):
+After you edited your `.env` configuration file on a bare metal/VM setup, you always need to execute the `composer dump-env` command (in Docker you just restart the containers).
+
+Running the `post-upgrade` script will also execute `composer dump-env` for you:
 
 ```bash
-composer dump-env prod
+./bin/post-upgrade
```
Comment (e-five256) [inline FAQ.md:132]: I'm not super knowledgeable on the command as a non-docker user but I assume there isn't a downside to running it when unneeded

**Review by e-five256 — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (melroy89) [reply .gitignore:20]: but that could potentially also ignore media-blabla files in the public folder right?

File: FAQ.md (hunk 2 above)
Comment (melroy89) [reply FAQ.md:132]: no there is no real down-side, maybe some minor cache clean-up, but in general there is no down side in running this command whenever you please.

File: .gitignore (hunk 1 above)
Comment (melroy89) [reply .gitignore:20]: or media.css .. just an example.

**Pull request merged**